storage/api/Storage.Samples/SetRPODefault.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_set_rpo_default]
+
+using System;
+using Google.Apis.Storage.v1.Data;
+using Google.Cloud.Storage.V1;
+
+public class SetRPODefaultSample
+{
+    public void SetRPODefault(
+        string bucketName = "your-unique-bucket-name")
```

Comment:
When it's just a few parameters, there's no need to put them on the next line.
```suggestion
    public void SetRPODefault(string bucketName = "your-unique-bucket-name")
```